lms/templates/ux/reference/teams-base.html

```diff
@@ -0,0 +1,165 @@
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/info">
+                 Course Info
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/discussion/forum">
+                 Discussion
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/course_wiki">
+                 Wiki
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/progress">
+                 Progress
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/instructor">
+                 Instructor
+             </a>
+          </li>
+
+    </ol>
+  </div>
+</nav>
+</%block>
+
+<%block name="headextra">
+<%static:css group='style-course-vendor'/>
+<%static:css group='style-course'/>
+<%block name="nav_skip">${"#seq_content" if section_title else "#course-content"}</%block>
+</%block>
+
+<%block name="js_extra">
+
+  <script type="text/javascript">
+    $(function(){
+        $(".ui-accordion-header a, .ui-accordion-content .subtitle").each(function() {
+          var elemText = $(this).text().replace(/^\s+|\s+$/g,''); // Strip leading and trailing whitespace
+          var wordArray = elemText.split(" ");
+          var finalTitle = "";
+          if (wordArray.length > 0) {
+            for (i=0;i<=wordArray.length-1;i++) {
+              finalTitle += wordArray[i];
+              if (i == (wordArray.length-2)) {
+                finalTitle += "&nbsp;";
+              } else if (i == (wordArray.length-1)) {
+                // Do nothing
+              } else {
+                finalTitle += " ";
+              }
+            }
+          }
+          $(this).html(finalTitle);
+        });
+      });
+  </script>
+
+</%block>
+
+<div class="container">
+  <div class="course-wrapper">
+    <section class="course-content" id="course-content">
+
+    <header class="page-header has-secondary">
+      <div class="header-main">
+        <nav class="breadcrumbs">
+          <a class="nav-item" href="">Sustainable Energy</a> <span class="icon fa-angle-right" aria-hidden="true"></span>
+          <a  class="nav-item" href="">Sample double nest</a> <span class="icon fa-angle-right" aria-hidden="true"></span>
+        </nav>
+        <h2 class="page-title">Solar Power</h2>
+        <p class="page-description">Teams working on projects related to Week 3 - 10 group activities</p>
+      </div>
+
+      <div class="header-secondary">
+        <div class="header-search search-teams">
+          <label for="search-teams" class="search-label">Search teams</label>
+          <input id="search-teams" class="search-field" placeholder="Search teams" />
+          <span class="icon fa-search" aria-hidden="true"></span>
```

Comment:
When previewing the UI for this, I thought this icon was a button or something interactive. I entered my search term and immediately thought due to its positioning that it was the submit button. If this was considered and ruled not an issue during the UI/VIZ design, cool. If not, should we revamp?